scaling_performance/network_optimization.adoc

```diff
@@ -170,6 +170,31 @@ network infrastructure.
 VXLAN-offload does not reduce latency. However, CPU utilization is reduced even
 in latency tests.
 
+[[scaling-performance-optimizing-mtu]]
+=== Optimizing the MTU for Your Network
+
+There are two MTUs (maximum transmission units) that matter: the network interface card (NIC) MTU, and the SDN overlay's MTU.
+
+The NIC MTU can be whatever you think is optimal for your network.
```

Comment:
@jeremyeder @knobunc @dcbw I'm not too sure what the jumbo frame info is trying to say. I consulted with a networking guy in the office and he said that it's perhaps not too relevant. Is there any of the info that's specific to OpenShift?

I put the other info in, though.